carma_cooperative_perception/config/cp_multiple_object_tracker_node.yaml

```diff
@@ -13,6 +13,7 @@ track_promotion_threshold: 3
 # its counter updates to this value
 # For example" if threshold is 10, a confirmed track will only be removed
 # after 10 "consecutive" non-occurrence to be removed
-track_removal_threshold: 10
+track_removal_threshold: 20
```

Comment:
This is needed for better tracking of pedestrian as they switch between different type of detection zones